docs/agents/android-agent.rst

```diff
@@ -17,6 +17,19 @@ Description
 An android agent that can be controlled via torques supplied to its joints.
 See :class:`~holodeck.agents.AndroidAgent` for more details.
 
+Control Schemes
+---------------
+ - **ANDROID_DIRECT_TORQUES** (```0```)
+A 94 dimensional vector of continuous values representing torques to be applied at each joint. There are 18 joints that can rotate in 3 dimensions, 10 that can rotate in 2 dimensions, and 20 that can only
```

Comment:
@jaydenmilne I wasn't exactly sure here. I don't think this is as concise or clear or accurate as it needs to be.